lexers/python.lua

```diff
@@ -33,7 +33,7 @@ lex:add_rule('constant', builtin_const + attr)
 local sq_str = lexer.range("'", true)
 local dq_str = lexer.range('"', true)
 local tq_str = lexer.range("'''") + lexer.range('"""')
-lex:add_rule('string', lex:tag(lexer.STRING, (S('fFrRbBrR') * S('rRfFrRbB') + S('ruRUfFbB'))^-1 *
+lex:add_rule('string', lex:tag(lexer.STRING, (S('fFrRbBrRtT') * S('rRfFrRbBtT') + S('ruRUfFbBtT'))^-1 *
```

Comment:
I honestly don't understand why this prefix pattern exists. I'm looking through the revision history and it looks like I added it here: https://github.com/orbitalquark/scintillua/commit/051d2ec00859bcf48c32f653309140dbf0e43e99#diff-46651a8c375626d6c358fc27f41c182bf025b6b95bb9435e2187fe6ab77775deR35, but I don't understand why I did that 😕 

This needs to be cleaned up before we can add the `t` prefix. Also, is a `T` prefix allowed? I couldn't find a reference to it in the PEP.

Reply:
I know I am late, but this is the canonical documentation: https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals .